tcl/pd_menus.tcl

```diff
@@ -575,6 +575,44 @@ proc ::pd_menus::create_preferences_menu {mymenu} {
         -command {::pd_menus::loadpreferences}
     $mymenu add command -label [_ "Forget All..."] \
         -command {::pd_menus::forgetpreferences}
+		
+	switch -- $::windowingsystem {
+       
+	   "win32" {
+	
+    $mymenu add command -label [_ "Reset .pd files to *this* Pd"] \
```

Comment:
I'm not sure whether this sentence makes sense to the 
Probably this is better:
> Always open .pd files with *this* installation of Pd."

This is a bit long for a menu-entry.
A better place might be a general purpose perferences-window.

Reply:
> This is a bit long for a menu-entry.
A better place might be a general purpose perferences-window.

We don't have that yet. :)